generators/server/templates/src/main/java/package/repository/UserRepository.java.ejs

```diff
@@ -173,7 +173,7 @@ public interface UserRepository extends <% if (databaseType === 'sql') { %>JpaRe
     <%_ if (databaseType === 'neo4j') { _%>
     <% if (!reactive) { %>// See https://github.com/neo4j/sdn-rx/issues/51<% } %>
     <% if (reactive) { %>Flux<% } else { %>List<% } %><<%= asEntity('User') %>> findAll();
-    
+
```

Comment:
I think query derivation is supported with this release. So the `@Query` annotations can be removed.
And `Pageable` should also supported in query derivation.

Reply:
It is. But I would like to fix tests before doing some cleanup: I think I've found a regression in Spring R2DBC, I'm trying to recreate a minimal project to illustrate the issue and report it.